speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/rppUtils/RppUtils.hpp

```diff
@@ -86,6 +87,14 @@ inline filesystem::path SingletonsFilePath()
     return {filesystem::temp_directory_path().append("Singletons")};
 }
 
+/// @brief        Adds an extra escape character in front of every backslash (\\) -> (\\\\)
+/// @param p_path The path to escape all the file separators for.
+/// @return       A string containing the newly escaped path.
+inline std::string Escape(const filesystem::path& p_path)
```

Comment:
I would rename this to AddEscapeCharacter or something similar, Escape can mean anything like escaping the program